src/main/java/ch/njol/skript/effects/EffScriptFile.java

```diff
@@ -77,20 +80,34 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 
 	@Override
 	protected void execute(Event event) {
+		OpenCloseable openCloseable = OpenCloseable.EMPTY;
+		RetainingLogHandler logHandler = null;
+		if (mark == RELOAD || mark == ENABLE) {
+			logHandler = new RetainingLogHandler().start();
+			openCloseable = logHandler;
+		}
 		if (scripts) {
 			for (Script script : scriptExpression.getArray(event)) {
 				@Nullable File file = script.getConfig().getFile();
-				this.handle(file, script.getConfig().getFileName());
+				this.handle(file, script.getConfig().getFileName(), openCloseable);
 			}
 		} else {
 			String name = scriptNameExpression.getSingle(event);
-			if (name == null)
-				return;
-			this.handle(ScriptLoader.getScriptFromName(name), name);
+			if (name != null)
+				this.handle(ScriptLoader.getScriptFromName(name), name, openCloseable);
+		}
+		if (logHandler != null) {
+			String[] errors = logHandler.getErrors().stream().map(LogEntry::getMessage).toArray(String[]::new);
+			if (mark == RELOAD) {
+				ExprScriptFileErrors.lastReloadedErrors = errors;
+			} else {
+				ExprScriptFileErrors.lastEnabledErrors = errors;
+			}
+			logHandler.printErrors();
```

Comment:
![image](https://github.com/user-attachments/assets/165e7f8d-a3f1-4f2e-9250-c5843fbbd66a)
This is not being handled properly